Maven Build Failure - API incompatibility in tycho-p2-director-plugin:2.7.3:materialize-products

### 👓 What did you see?

Maven Build Fails when running `maven clean install` attempting to build locally

### ✅ What did you expect to see?

Maven Build should run successfully and create the local updatesite

### 📦 Which tool/library version are you using?
Java 11
maven 3.6.3

### 🔬 How could we reproduce it?

Clone the repository
Run `maven clean install`

See the error log
``
[INFO] Reactor Summary for Cucumber Eclipse Parent 2.0.0-SNAPSHOT:
[INFO] 
[INFO] Cucumber Eclipse Parent ............................ SUCCESS [  0.105 s]
[INFO] [target] targetdefinition .......................... SUCCESS [  0.176 s]
[INFO] [bundle] Cucumber Editor ........................... SUCCESS [  2.593 s]
[INFO] [bundle] Cucumber Eclipse Plugins .................. SUCCESS [  2.760 s]
[INFO] [bundle] Java ...................................... SUCCESS [  1.503 s]
[INFO] [feature] third party dependencies ................. SUCCESS [  0.538 s]
[INFO] [feature] Cucumber Eclipse ......................... SUCCESS [  0.153 s]
[INFO] [product] Cucumber Eclipse ......................... FAILURE [ 19.224 s]
[INFO] [updatesite] io.cucumber.eclipse.updatesite.eclipse-repository SUCCESS [ 19.154 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:35 min (Wall Clock)
[INFO] Finished at: 2022-05-24T16:02:52+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-p2-director-plugin:2.7.3:materialize-products (materialize-products) on project cucumber-eclipse: Execution materialize-products of goal org.eclipse.tycho:tycho-p2-director-plugin:2.7.3:materialize-products failed: An API incompatibility was encountered while executing org.eclipse.tycho:tycho-p2-director-plugin:2.7.3:materialize-products: java.lang.NoSuchMethodError: 'void org.eclipse.tycho.p2.tools.director.shared.DirectorRuntime$Command.setProfileProperties(java.util.Map)'
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.eclipse.tycho:tycho-p2-director-plugin:2.7.3
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/gayan/.m2/repository/org/eclipse/tycho/tycho-p2-director-plugin/2.7.3/tycho-p2-director-plugin-2.7.3.jar
[ERROR] urls[1] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-component-annotations/2.1.1/plexus-component-annotations-2.1.1.jar
[ERROR] urls[2] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-archiver/4.2.7/plexus-archiver-4.2.7.jar
[ERROR] urls[3] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-utils/3.4.1/plexus-utils-3.4.1.jar
[ERROR] urls[4] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-io/3.2.0/plexus-io-3.2.0.jar
[ERROR] urls[5] = file:/home/gayan/.m2/repository/commons-io/commons-io/2.6/commons-io-2.6.jar
[ERROR] urls[6] = file:/home/gayan/.m2/repository/org/iq80/snappy/snappy/0.4/snappy-0.4.jar
[ERROR] urls[7] = file:/home/gayan/.m2/repository/org/tukaani/xz/1.9/xz-1.9.jar
[ERROR] urls[8] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.26/plexus-interpolation-1.26.jar
[ERROR] urls[9] = file:/home/gayan/.m2/repository/org/codehaus/plexus/plexus-cipher/2.0/plexus-cipher-2.0.jar
[ERROR] urls[10] = file:/home/gayan/.m2/repository/org/eclipse/platform/org.eclipse.osgi/3.17.200/org.eclipse.osgi-3.17.200.jar
[ERROR] urls[11] = file:/home/gayan/.m2/repository/org/eclipse/tycho/sisu-equinox-launching/2.7.3/sisu-equinox-launching-2.7.3.jar
[ERROR] urls[12] = file:/home/gayan/.m2/repository/org/apache/commons/commons-exec/1.3/commons-exec-1.3.jar
[ERROR] urls[13] = file:/home/gayan/.m2/repository/org/eclipse/tycho/tycho-bundles-external/2.7.3/tycho-bundles-external-2.7.3.zip
[ERROR] urls[14] = file:/home/gayan/.m2/repository/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
[ERROR] urls[15] = file:/home/gayan/.m2/repository/org/apache/commons/commons-compress/1.21/commons-compress-1.21-sources.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>io.cucumber.eclipse:parent:2.0.0-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :cucumber-eclipse
``

--- a/pom.xml
+++ b/pom.xml
@@ -8,7 +8,7 @@
 	<name>Cucumber Eclipse Parent</name>
 
 	<properties>
-		<tycho-version>2.3.0</tycho-version>
+		<tycho-version>2.7.3</tycho-version>
 		<maven-scm-plugin-version>1.11.1</maven-scm-plugin-version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.source>11</maven.compiler.source>

